tests/test_ball_detector.py

```diff
@@ -73,6 +79,12 @@ def test_detect_one_ball(self):
         # Check ball radii
         self.assertAlmostEqual(red_ball.radius, ball_radius, delta=self._MAX_DIMENSION_DIFFERENCE)
 
+        # Check angle
+        self.assertAlmostEqual(red_ball.angle, red_ball_angle, delta=self._MAX_DIMENSION_DIFFERENCE)
+
+        # Check center points deque has been appended
+        self.assertEqual(len(red_ball.center_points_cv), 1)
```

Comment:
Why is it called `center_points_cv`? They are co-ordinates and have nothing to do with OpenCV?